code/__defines/misc.dm

```diff
@@ -302,6 +302,7 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 #define MIDNIGHT_ROLLOVER		864000	//number of deciseconds in a day
 
 #define DEBUG_REF(D) (D ? "\ref[D]|[D] ([D.type])" : "NULL")
+#define PROCLOG(thing) log_debug("[THIS_PROC_TYPE_WEIRD]: [thing]")
```

Comment:
`THIS_PROC_TYPE_WEIRD` doesn't work for all procs IIRC.

Maybe make `PROCLOG` and `PROCLOG_WEIRD`?